recipes/suitcase-csv/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "suitcase-csv" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "2f8eece9502084dcd81143e8ff95ad15ead43325f2fc063f0642789ffecd8a8f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Let's synchronize the commands used across the recipes. I'd suggest using the one from the databroker-pack's recipe, that also matches the recommendation from https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip (I excluded double-quotes as the auto-tick bot does that anyway):

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```